resolwe/flow/executors/__init__.py

```diff
@@ -89,7 +97,7 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
-        dir_mode = settings.FLOW_EXECUTOR.get('DATA_DIR_MODE', 0o755)
+        dir_mode = getattr(settings, 'FLOW_EXECUTOR["DATA_DIR_MODE"]', 0o755)
```

Comment:
In [1]: class A(object):
       ...:     pass
       ...:

    In [2]: a = A()

    In [3]: a.b = {'c': 1}

    In [4]: getattr(a, 'b["c"]', 0)
    Out[4]: 0

This is not working as expected.